### PR TITLE
Workaround for collectd problem

### PR DESCRIPTION
**Pull request by juvenal**
# Motivation
From some time now I had stumbled on a problem with the collectd listener, receiving some metric that crashed the whole influxdb backend due to &name == nil. My asset network runs a lot of different machines, which implies that not every collectd are on the same version number (and even not with the same plugins), although I keep them at least within the compatibility limits. I know that this may be partly responsible for the issue that I found.
In an attempt to make the code somewhat safer in this regard (I think that this multiple collectd versions situation may happen with others), I patched this workaround and had tested it for some time now (confirming that it's working properly) prior to send it as a PR.

## Commit message
This commit adds a workaround for an error found on the collectd service.go
code, where &name == nil crashed the whole influxdb backend. It also fixes a typo
on the indentation of package.sh and updates the Go version to use on package.sh
build script.

### CODE REVIEW

**Comment by jwilder** [comment]
What specific error are you seeing?  

It is really helpful to provide the information specified here: https://github.com/influxdata/influxdb/blob/master/CONTRIBUTING.md#bug-reports when submitting bug reports or changes.

File: package.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ if [ -z "$FPM" ]; then
     FPM=`which fpm`
 fi
 
-GO_VERSION="go1.4.3"
```
Comment (jwilder) [inline package.sh]: We can't use go 1.5.  See #5217.  Please revert the changes to this file.

File: services/collectd/service.go [hunk 2]
```diff
@@ -299,7 +299,12 @@ func (s *Service) UnmarshalCollectd(packet *gollectd.Packet) []models.Point {
 		p, err := models.NewPoint(name, tags, fields, timestamp)
 		// Drop invalid points
 		if err != nil {
-			s.Logger.Printf("Dropping point %v: %v", name, err)
```
Comment (jwilder) [inline services/collectd/service.go]: `name` == nil should not trigger here panic.  There was a panic in this code that was fixed via https://github.com/influxdata/influxdb/commit/01193668cf88658e3dc99a2f2274503b3cece989

Can you provide the stack trace and version of the server your are running?

**Comment by juvenal** [comment]
Jason, I checked all your comments, and I'm really sorry to waste your time. It seems that my changes were useless, as the real fix was already written 12 days ago via [0119366](https://github.com/influxdata/influxdb/commit/01193668cf88658e3dc99a2f2274503b3cece989) as you said, and I completely missed issue #5217 regarding Go version 1.5 and up at the moment.
Just for the record of this, I'm running InfluxDB on a Raspberry Pi 2 (ARM) to record metrics from all my internal network assets using collectd network reader, and I was using some early 0.10.0 (really a clone of the master branch at the time). I completely missed the slight change (and much wiser one) used on the real fix (switching p.Name by name) on the log call. If you still need some more information about this failed and useless PR, please feel free to ask.
I'm closing this PR, as it's useless...
Again, I'm sorry to take your time on this.

**Comment by jwilder** [comment]
No problem @juvenal. I'm glad your issues are fixed.  Thanks!
